### PR TITLE
updated colors and images

### PR DESCRIPTION
**Pull request by macfarla**
Signed-off-by: Sally MacFarlane <sally.macfarlane@consensys.net>

### CODE REVIEW

File: src/constants/colors.ts [hunk 1]
```diff
@@ -2,15 +2,15 @@ import { tint, shade, readableColor } from 'polished';
...
-  green: '#28C081',
+  green: '#01D49A',
   yellow: '#FD9D28',
   red: '#DC2C10',
-  blurple: '#4E3FCE',
-  consensysblue: '#3259D6',
-  pegasys: '#25D78F'
+  blurple: '#CCD8E1',
+  consensysblue: '#2C56DD',
+  pegasys: '#2C56DD'
```
Comment (mark-terry) [inline src/constants/colors.ts:13]: Does this need a name change?

**Review by mark-terry — COMMENTED** [review]
One query.

File: src/constants/colors.ts (hunk 1 above)
Comment (macfarla) [reply src/constants/colors.ts]: changed to protocol

**Review by mark-terry — APPROVED** [review]
LGTM.

**Pull request merged**